homeassistant/components/homekit/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "homekit",
   "name": "HomeKit",
   "documentation": "https://www.home-assistant.io/integrations/homekit",
-  "requirements": ["HAP-python==2.8.3","fnvhash==0.1.0","PyQRCode==1.2.1","base36==0.1.1"],
+  "requirements": ["HAP-python==2.8.3","fnvhash==0.1.0","PyQRCode==1.2.1","base36==0.1.1","cffi==1.14.0","jpegtran-cffi==0.5.2"],
```

Comment:
I think that we should use pillow, as we have use that in other integrations too

Reply:
I tried pillow first, it was too slow and ended up making the watch stall and spin.  Then opencv which was too big.   So ended up here because performance was important to get it stable.

Reply:
@pvizeli do you know if this will work on our alpine images?

Reply:
@balloob Might be moot now.  I switched it out to `PyTurboJPEG` and it now falls back to previous slow behavior if `libjpegturbo` is missing (its was install on alpine image I tested)

Reply:
```
# docker exec -it hass-test /bin/bash
bash-5.0# cd /usr/lib
bash-5.0# ls *jpeg*
libjpeg.so.8           libjpeg.so.8.2.2       libturbojpeg.so.0      libturbojpeg.so.0.2.0
```

Reply:
If this alternative is workable I’ll write tests for it tomorrow

Reply:
I agree with Paulus, we should use pillow which is basically just a wrapper around the library like you archive to use. Our pillow on Home Assistant support libjpegturbo over the standard jpeg library. But you are right, if you use a venv, pillow could be slower depend on how do you compile it.

Anyway, I'm also fine with PyTurboJPEG - however we use pillow on a lot of place and maybe we should look that this library going to be fix if there is an issue instead to maintain 15 different ways in our code base.

Reply:
Edit, sorry - I missed Paulus second question. Yeah PyTurboJPEG and other ctype based library work on our image and only in our image because we patch the python for this: https://github.com/home-assistant/docker-base/blob/master/python/3.7/musl-find_library.patch

Reply:
Thanks for looking @pvizeli `pillow` was definitely the first choice, unfortunately it didn't fit the use case because it was at least an order of magnitude slower. 

<img width="1124" alt="Screen Shot 2020-05-07 at 9 28 41 AM" src="https://user-images.githubusercontent.com/663432/81306492-28ceb500-9045-11ea-9e62-fc7077698f18.png">